### DISCUSSION TITLE
Is there any command to show the path of wezterm's config file? (Need to find wezterm.lua on Windows)

### DISCUSSION THREAD

**Discussion by ivaquero** [wezterm/wezterm#3418] [Q&A] [answered]
I install wezterm on win11 using scoop package manager. But modified the config file in the persist folder, but it doesn't work, moreover there is no `%HOME%/.wezterm.lua` that is said in the docs.

Where is the wezterm.lua on Windows? How can I find it?

**Comment by wez** [comment, marked as answer]
You need to create it. As the docs say, creating it at `%HOME%/.wezterm.lua` is recommended.

**Comment by ivaquero** [comment]
Thank you, it works!
